src/azure-cli/azure/cli/command_modules/serviceconnector/action.py

```diff
@@ -114,7 +119,7 @@ def get_action(self, values, option_string, command_name):  # pylint: disable=no
                 d['client_id'] = v[0]
             elif kl == 'subs-id':
                 d['subscription_id'] = v[0]
-            elif is_mysql_target(command_name) and kl == 'mysql-identity-id':
+            elif is_mysql_target(command_name) and not is_k8s_source(command_name) and kl == 'mysql-identity-id':
```

Comment:
will passwordless cmds fail? I remember there's  some change need in passwordless extension before aks +passwordless is fully tested. could you pls sync with Xiaofan to make sure passwordless cmds fail with not supported error before ready? then aks +passwordless support.